packages/app/src/systems/Core/components/Layout/TopBar.tsx

```diff
@@ -22,24 +22,32 @@ import { useNetworks } from '~/systems/Network/hooks';
 import { NetworkScreen } from '~/systems/Network/machines';
 import { Sidebar } from '~/systems/Sidebar';
 
+export enum TopBarType {
+  internal,
+  external,
+}
+
 type TopBarProps = {
+  type?: TopBarType;
   onBack?: () => void;
   children?: ReactNode;
 };
 
-export function TopBar({ onBack }: TopBarProps) {
+// ----------------------------------------------------------------------------
```

Comment:
Change this to JSDoc Adding something like this

```jsx
/**
 * @description - TopBar used inside Application
 */
```

Reply:
We don't auto-generate docs for now don't know if is needed